### ISSUE TITLE
Installing older version of ggmap on R 3.6.1

### ISSUE THREAD

**Issue by AyeMiranda** [daroczig/CEU-DV2#4]
Hi @daroczig , I'm trying to install the old version of `ggmap` using this code: 
`install.packages('https://cran.r-project.org/src/contrib/Archive/ggmap/ggmap_2.6.2.tar.gz')`
 but it returns this warning and wouldn't install:
```
Warning in install.packages :
  package ‘https://cran.r-project.org/src/contrib/Archive/ggmap/ggmap_2.6.2.tar.gz’ is not available (for R version 3.6.1)
```

May I know which R version is recommended for this class to avoid some compatibility issues with the packages we will be working with? There's a newer version (3.6.2) released 2019-12-12, should we use this (or some older version, since we are also using an old version of ggmap)?

Thanks!

**Comment by daroczig** [comment]
You are on Windows?

I've just realized that running the above `install.packages` command might not be that trivial as it is compiling the package instead of downloading the binary -- that is preferred on Windows and Mac.

A workaround could be installing R tools on Windows as per https://cran.r-project.org/bin/windows/Rtools/ and retrying.

Or probably even better to rely on previously built binaries -- can you please give the below a try?

```r
install.packages('ggmap', repos = 'https://cran.microsoft.com/snapshot/2019-02-01/')
```

**Comment by AyeMiranda** [comment]
I'm using Mac (sorry forgot to mention earlier). Here are my details:
R version 3.6.1 (2019-07-05)
Platform: x86_64-apple-darwin15.6.0 (64-bit)
Running under: macOS Mojave 10.14.5

This code worked for me:
`install.packages('ggmap', repos = 'https://cran.microsoft.com/snapshot/2019-02-01/')
 `

After installing, I tried `geocode('Budapest, Nador utca 9', source = 'dsk', )` and I was able to fetch the data.

Thanks a lot! 👍
